lms/static/sass/elements/_system-feedback.scss

```diff
@@ -145,9 +145,955 @@
+
+  &.wrapper-alert-warning {
+    box-shadow: 0 1px 1px $white, inset 0 2px 2px $shadow-d1, inset 0 -4px 1px $orange;
+
+    .feedback-symbol {
+      color: $orange;
+    }
+  }
+
+  &.wrapper-alert-error {
+    box-shadow: 0 1px 1px $white, inset 0 2px 2px $shadow-d1, inset 0 -4px 1px $red-l1;
+
+    .feedback-symbol {
+      color: $red-l1;
+    }
+  }
+
+  &.wrapper-alert-confirmation {
+    box-shadow: 0 1px 1px $white, inset 0 2px 2px $shadow-d1, inset 0 -4px 1px $green;
+
+    .feedback-symbol {
+      color: $green;
+    }
+  }
+
+  &.wrapper-alert-announcement {
+    box-shadow: 0 1px 1px $white, inset 0 2px 2px $shadow-d1, inset 0 -4px 1px $blue;
+
+    .feedback-symbol {
+      color: $blue;
+    }
+  }
+
+  &.wrapper-alert-step-required {
+    box-shadow: 0 1px 1px $white, inset 0 2px 2px $shadow-d1, inset 0 -4px 1px $pink;
+
+    .feedback-symbol {
+      color: $pink;
+    }
+  }
+}
+
+// adopted alerts
+.alert {
+  @include box-sizing(border-box);
+  @include clearfix();
+  margin: 0 auto;
+  width: flex-grid(12);
+  max-width: $fg-max-width;
+  min-width: $fg-min-width;
+  color: $white;
+
+  strong {
+    @extend %t-strong;
+  }
+
+  .feedback-symbol, .copy {
+    float: left;
+  }
+
+  .feedback-symbol {
+    @include transition (color $tmg-f1 ease-in-out 0s);
+    @extend %t-icon3;
+    width: flex-grid(1, 12);
+    margin: ($baseline/4) flex-gutter() 0 0;
+    text-align: right;
+  }
+
+  .copy {
+    width: flex-grid(10, 12);
+    margin-top: ($baseline/2);
+    color: $gray-l2;
+
+    .title {
+      @extend %t-title7;
+      margin-bottom: 0;
+      color: $white;
+    }
+  }
+
+  // with actions
+  &.has-actions {
+
+    .feedback-symbol {
+      width: flex-grid(1, 12);
+    }
+
+  	.copy {
+  		width: flex-grid(7, 12);
+  		@include margin-right(flex-gutter());
+  	}
+
+  	.nav-actions {
+  	  width: flex-grid(4, 12);
+      @include float(left);
+      margin-top: ($baseline/2);
+  	  text-align: right;
+
+  	  .nav-item {
+  	    display: inline-block;
+  	    vertical-align: middle;
+  	    @include margin-right($baseline/2);
+
+  	    &:last-child {
+  	      @include margin-right(0);
+  	    }
+
+       .action-primary {
+        @extend %t-action4;
+        @extend %t-strong;
+      }
+
+      .action-secondary {
+        @extend %t-action4;
+      }
+    }
+  }
+}
+
+  // with cancel
+  .action-alert-close {
+    @include border-bottom-radius(($baseline/5));
+    position: absolute;
+    top: -($baseline/10);
+    right: $baseline;
+    padding: ($baseline/4) ($baseline/2) 0 ($baseline/2);
+    background: $gray-d4;
+    text-align: center;
+
+    .label {
+      @extend %cont-text-sr;
+    }
+
+    .icon {
+      @extend %t-icon6;
+      color: $white;
+      width: auto;
+      margin: 0;
+      padding: 2px;
+    }
+
+    &:hover {
+      background: $gray-d1;
+    }
+  }
+
+  // with dismiss (to sunset action-alert-clos)
+  .action-dismiss {
+
+    .button {
+      @extend %btn-secondary-white !optional;
```

Comment:
sounds great. would you mind adding a  comment to this effect here to clarify why optional was used? I can take a look in a later PR about moving the dependency over to LMS as well. thanks!